Adding new --config argument to zimit

Since zimit has fully migrated to use the [webrecorder/browsertrix-crawler](https://github.com/webrecorder/browsertrix-crawler) it would be nice to have more available options for the crawler. For the first step adding a `--config` option would be awesome for all advanced use cases, so you can configure [webrecorder/browsertrix-crawler](https://github.com/webrecorder/browsertrix-crawler) to fit your needs (e.g. blockRules, etc.) via a given [yaml file](https://github.com/webrecorder/browsertrix-crawler#yaml-crawl-config).

Unfortunately currently any other from zimit unsuported options are rejected with the error:

```
zimit: error: unrecognized arguments: --blockMessage --blockRules
Error: Process completed with exit code 2.
```

--- a/zimit.py
+++ b/zimit.py
@@ -317,6 +317,12 @@ def zimit(args=None):
         help="If set, output stats as JSON to this file",
     )
 
+    parser.add_argument(
+        "--config",
+        help="Path to YAML config file. If set, browsertrix-crawler will use this file"
+        "to configure the crawling behaviour if not set via argument.",
+    )
+
     zimit_args, warc2zim_args = parser.parse_known_args(args)
 
     # pass url and output to warc2zim also
@@ -488,6 +494,7 @@ def get_node_cmd_line(args):
         "timeLimit",
         "healthCheckPort",
         "overwrite",
+        "config",
     ]:
         value = getattr(args, arg)
         if value:

